Mathlib/CategoryTheory/Elements.lean

```diff
@@ -136,6 +136,16 @@ def map {F₁ F₂ : C ⥤ Type w} (α : F₁ ⟶ F₂) : F₁.Elements ⥤ F₂
 theorem map_π {F₁ F₂ : C ⥤ Type w} (α : F₁ ⟶ F₂) : map α ⋙ π F₂ = π F₁ :=
   rfl
 
+variable {D : Type u} [Category.{v} D] in
+
```

Comment:
```suggestion
variable {D : Type*} [Category D] in

```
Alternatively, declare new universes `v' u'` and use `{D : Type u'} [Category.{v'} D]`.